e2e/fixtures.js

```diff
@@ -16,7 +23,32 @@ function consoleMessageToString(msg) {
     at (${url} ${lineNumber}:${columnNumber})`;
 }
 
+//The following is based on https://github.com/mxschmitt/playwright-test-coverage
+const istanbulCLIOutput = path.join(process.cwd(), '.nyc_output');
+
+function generateUUID() {
+    return crypto.randomBytes(16).toString('hex');
```

Comment:
we should use this guy instead:
https://github.com/nasa/openmct/blob/master/package.json#L67